sound/soc/sof/trace.c

```diff
@@ -144,6 +149,9 @@ int snd_sof_init_trace_ipc(struct snd_sof_dev *sdev)
 	params.buffer.size = sdev->dmatb.bytes;
 	params.buffer.pages = sdev->dma_trace_pages;
 	params.stream_tag = 0;
+	shift = jiffies_64 - sdev->bootup_timestamp;
+	/* jiffies to ms */
+	shift = shift * 1000 / HZ;
```

Comment:
Why do you want to represent the offset in ms, the firmware time resolution is based on the oscillator (42ns and lower). The shift should not be quantized in ms?

Reply:
@plbossart The resolution of jiffies is based on the value of HZ which typically is 1000. I don't like to quantized it but it is ms originally.